### PR TITLE
Updating Sartosa

### PR DESCRIPTION
**Pull request by Mjb141**
Updating Sartosa after playing a campaign.

### CODE REVIEW

File: src/db/effect_bundles_to_effects_junctions_tables_vco.tsv [hunk 1]
```diff
@@ -1,99 +1,100 @@
...
-vco_victory_payload_cst_sar_2_daughter_of_manann_1	wh2_main_effect_force_stat_melee_attack_at_sea	12.0	faction_to_force_own_unseen	start_turn_completed
-vco_victory_payload_cst_sar_2_daughter_of_manann_1	wh2_main_effect_force_stat_leadership_at_sea	12.0	faction_to_force_own_unseen	start_turn_completed
-vco_victory_payload_cst_sar_2_daughter_of_manann_2	vco_cst_rotting_units_upkeep_reduction	50.0	faction_to_force_own_unseen	start_turn_completed
-vco_victory_payload_cst_sar_3_dead_man_s_vengeance_1	vco_cst_zombie_deckhands_missile_strength	20.0	faction_to_force_own_unseen	start_turn_completed
-vco_victory_payload_cst_sar_3_dead_man_s_vengeance_2	vco_cst_necrofex_colossus_upkeep_reduction	33.0	faction_to_force_own_unseen	start_turn_completed
-vco_victory_payload_cst_sar_3_dead_man_s_vengeance_2	vco_cst_necrofex_colossus_recruitment_time_reduction	1.0	faction_to_faction_own_unseen	start_turn_completed
+vco_campaign_victory_dummy	vco_campaign_victory_dummy	1.0000	faction_to_faction_own_unseen	start_turn_completed
```
Comment (Mjb141) [inline src/db/effect_bundles_to_effects_junctions_tables_vco.tsv:3]: RFPM update has changed how floating values are displayed...

**Comment by Mjb141** [comment]
I played a 150-turn Sartosa campaign and these points stood out as requiring updates:

* 15000 from raiding felt like a chore before I even hit 5000 from raiding. Raiding generates no infamy but sacking/razing do, so why waste turns raiding? Raiding also prevents you from digging that turn, which is a vampire coast faction mechanic. Why would we lock the player out of that option?

* 20000 infamy is a slog. It's definitely not a 50-70 turn route considering it's shared with the above objective. 200 per raze and some measly gains per turn (think 20-50 by early-mid game). Sartosa play tall (fewer settlements, fewer armies early on until cove/sacking income picks up) with a focus on shipbuilding capable lords. The only way to make 20000 infamy in 50-70 turns is trying to play like you're Khorne/Taurox but you lack all their mechanics to do that.

* +20% missile strength for units you mostly want to ditch beyond early game is terrible. They have low missile strength anyway and 20% increase on a low number is... a low number. I actively hated having deckhands in my army beyond the early "must use cheap inexpensive units" part of the game. This reward is to be replaced entirely with Deck Gunner buffs, a tier 4 unit that you will want to keep in your armies.

* I still want to find a way to get ship growth in as a reward somewhere. Hiring shipbuilding lords in mid game is utterly painful - you have to basically wait till they can grow themselves into usefulness slowly, or have them follow another shipbuilding lord (read: Aranessa) to recruit from which defeats the point of hiring multiple shipbuilding lords entirely. Did not enjoy this at all. Sartosa is almost always economy/defensive/landmark buildings only, no military buildings due to not enough slots.

These points stood out as good areas:

* Route 3 is short and sweet, but I found that acceptable because routes 1 and 2 are a bit longer.

* Rewarding infamy in routes 2 and 3 is both good and necessary - keep it.

**Comment by Mjb141** [comment]
Sartosa needs docs adding (she was implemented prior to 'docs-as-PRs' starting). I'll add them once this PR is 1) done and 2) approved and merged

**Comment by msolefonte** [comment]
Once a PR is actually ready for review, please mark it as such. I think this one is good to go.

**Comment by Mjb141** [comment]
Needs docs adding, will do late this evening and mark this ready at that time. It's beta-7 anyway, we have time to do this one after 6.

**Comment by Mjb141** [comment]
Docs + changelog added, marking as ready for review

**Review by msolefonte — APPROVED** [review]

**Comment by msolefonte** [comment]
Looks good to me.

**Comment by Mjb141** [comment]
Docs updated.

We'll deal with RFPM's update for this beta.

**Review by msolefonte — APPROVED** [review]
Testers have fun 😎

**Pull request merged**